plugins/com.google.cloud.tools.eclipse.appengine.localserver/src/com/google/cloud/tools/eclipse/appengine/localserver/server/LocalAppEngineServerLaunchConfigurationDelegate.java

```diff
@@ -299,6 +299,22 @@ DefaultRunConfiguration generateServerRunConfiguration(ILaunchConfiguration conf
       devServerRunConfiguration.setAdditionalArguments(programArguments);
     }
 
+    boolean environmentAppend =
+        configuration.getAttribute(ILaunchManager.ATTR_APPEND_ENVIRONMENT_VARIABLES, true);
+    if (!environmentAppend) {
+      throw new CoreException(StatusUtil.error(this, "'Replace environment' not yet supported"));
```

Comment:
Can the error message more specific to point out that the _Replace environment_ is a checkbox in the _Environment_ launch tab? Users will hit the error far later when actually executing the dev appserver, so users can get easily lost about what this means.

Ideally, the _Replace environment_ checkbox should be disabled, but I guess this is difficult to implement?

Reply:
Unlike some of the other tabs, it is possible but it's brittle (fragile superclass and all).  I forgot to open an issue with appengine-plugins-core to add support for append/replace (https://github.com/GoogleCloudPlatform/appengine-plugins-core/issues/446).

Reply:
> Can the error message be more specific to point out that the _Replace environment_ is a checkbox in the _Environment_ launch tab?

Can we still do this? I think the current error message is too distance from the actual error source.

Reply:
oops, will do